source/includes/connection-snippets/scram-snippets/php-connection.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/* Begin scram connection */
+$client = new MongoDB\Client(
+    'mongodb+srv://<username>:<password>@<cluster-url>/test?retryWrites=true&w=majority'
+);
+
+$collection = $client->testDB->testCol;
+$docCount = $collection->count([]);
```

Comment:
[`count()`](https://docs.mongodb.com/php-library/current/reference/method/MongoDBCollection-count/) is deprecated. Please use [`countDocuments()`](https://docs.mongodb.com/php-library/current/reference/method/MongoDBCollection-countDocuments/). Also, note that the filter argument is optional defaults to `[]`, so you can simply call `$collection->countDocuments()`.